docs/iris/gallery_code/meteorology/plot_lagged_ensemble.py

```diff
@@ -116,42 +115,29 @@ def main():
 
     # set a global title for the postage stamps with the date formated by
     # "monthname year"
-    plt.suptitle(
-        "Surface temperature ensemble forecasts for %s"
-        % (time.strftime("%B %Y"),)
-    )
+    title = "Surface temperature ensemble forecasts for {}"
+    plt.suptitle(title.format(time.strftime("%B %Y")))
 
     iplt.show()
 
     # -------------------------------------------------------------------------
     # Plot #2: ENSO plumes
     # -------------------------------------------------------------------------
 
-    # Nino 3.4 lies between: 170W and 120W, 5N and 5S, so define a constraint
-    # which matches this
-    nino_3_4_constraint = iris.Constraint(
-        longitude=lambda v: -170 + 360 <= v <= -120 + 360,
-        latitude=lambda v: -5 <= v <= 5,
+    # Nino 3.4 lies between: 170W and 120W, 5N and 5S, so use the intersection
+    # method to restrict to this region.
+    nino_cube = surface_temp.intersection(
+        latitude=[-5, 5], longitude=[-170, -120]
     )
 
-    nino_cube = surface_temp.extract(nino_3_4_constraint)
-
-    # Subsetting a circular longitude coordinate always results in a circular
-    # coordinate, so set the coordinate to be non-circular
-    nino_cube.coord("longitude").circular = False
```

Comment:
Not really sure why this mattered...?

Reply:
Hmm I strongly suspect it is to do with how the plot was rendered around/over the dateline,

Reply:
But then we're taking an area mean before we plot it... 🤷